src/axelar/AxelarTransceiver.sol

```diff
@@ -11,27 +11,32 @@ import {IAxelarGasService} from "@axelar-network/axelar-gmp-sdk-solidity/contrac
 import {
     StringToAddress, AddressToString
 } from "@axelar-network/axelar-gmp-sdk-solidity/contracts/libs/AddressString.sol";
-import { Upgradable } from "@axelar-network/axelar-gmp-sdk-solidity/contracts/upgradable/Upgradable.sol";
+// Use their libraries
+import { Transceiver } from "@wormhole-foundation/native_token_transfer/Transceiver/Transceiver.sol";
 
-import {Managed} from "./utils/Managed.sol";
 import { IAxelarTransceiver } from './interfaces/IAxelarTransceiver.sol';
 
-contract AxelarTransceiver is IAxelarTransceiver, AxelarExecutable, Managed, Upgradable {
+contract AxelarTransceiver is IAxelarTransceiver, AxelarExecutable, Transceiver {
     IAxelarGasService public immutable gasService;
 
     // These mappings are used to convert between chainId and chainName as Axelar accept chainName as string format
-    mapping(uint16 => string) public idToAxelarChainIds;
-    mapping(string => uint16) public axelarChainIdToId;
-    mapping(uint16 => string) public idToAxelarAddress;
-    mapping(string => uint16) public axelarAddressToId;
+    struct AxelarTransceiverStorage {
```

Comment:
suggest breaking this up into separate storage slots. loading all the mappings at each use site may not be gas optimal.

Reply:
It does not, storage is only loaded when you access something with `slot. axelarChainIdToId[...]`

Reply:
we're using [this](https://eips.ethereum.org/EIPS/eip-7201) storage pattern that's also being used by OZ upgradable contracts from v5.